README.md

```diff
@@ -53,11 +53,12 @@ And render it:
 
 ```php
 <?php
-$m = new Mustache_Engine;
+$m = new Mustache_Engine(array('entity_flags' => ENT_QUOTES));
 $chris = new Chris;
 echo $m->render($template, $chris);
 ```
 
+*Note:* we recommend using `ENT_QUOTES` as a default of [entity_flags](https://github.com/bobthecow/mustache.php/wiki#entity_flags) for security reasons.
```

Comment:
How about “...to decrease the chance of X” rather than “... for security reasons”?

Reply:
Done. Please take a look.